src/main/resources/sam.conf

```diff
@@ -177,6 +177,17 @@ azureServices {
   managedAppClientId = ${?AZURE_MANAGED_APP_CLIENT_ID}
   managedAppClientSecret = ${?AZURE_MANAGED_APP_CLIENT_SECRET}
   managedAppTenantId = ${?AZURE_MANAGED_APP_TENANT_ID}
-  managedAppPlanIds = ["terra-dev", "terra-workspace-dev-plan"]
+  managedAppPlans = [
+    {
+        name = "terra-dev"
+        publisher = "thebroadinstituteinc1615909626976"
+        authorizedUserKey = authorizedTerraUser
```

Comment:
`authorizedUserKey` was added mimicking how BPM does it. Sam needs to do this check as well to keep users from doing mean things by calling Sam directly.